components/encryption/src/manager/mod.rs

```diff
@@ -691,14 +691,12 @@ impl DataKeyManager {
 
 impl Drop for DataKeyManager {
     fn drop(&mut self) {
-        self.rotate_terminal
-            .send(())
-            .expect("DataKeyManager drop send");
-        self.background_worker
-            .take()
-            .expect("DataKeyManager worker take")
-            .join()
-            .expect("DataKeyManager worker join");
+        if let Err(e) = self.rotate_terminal.send(()) {
+            info!("failed to terminate background rotation, are we shutting down?"; "err" => %e);
```

Comment:
Previously this was a panic? Shouldn't it at least be logged at the error level?

Reply:
Yes, it's panic in the past. This happens when shutting down, so may not be an error at all.